### PR TITLE
repliaction depth user education improvements v3.10

### PR DESCRIPTION
**Pull request by MaxDiz**
updates to Reports, Contacts, and Users pages to coincide with replication depth improvement made in v3.10 - https://github.com/medic/cht-docs/pull/58

closes: https://github.com/medic/cht-docs/issues/226

### CODE REVIEW

File: content/en/apps/features/contacts/_index.md [hunk 1]
```diff
@@ -10,19 +10,16 @@ relatedContent: >
...
-  apps/tutorials/contact-and-users-2
   apps/examples/contact-tracing
 ---
 <!-- ## Contacts: Person and Family Profiles -->
 <!-- TODO Refine screenshots, and add desktop view. -->
 
 “People” is the generic name used for individuals in apps built with the Core Framework. They can be patients, family members, nurses or health workers. Anyone with a profile in your app is a person.
 
-“Places” is the generic name that represents a level in the hierarchy. “People” belong to “places” and “places” belong to other higher level “places” in the hierarchy.
+“Places” is the generic name that represents a level in the [hierarchy]({{< ref "apps/concepts/hierarchy" >}}). “People” belong to “places” and “places” belong to other higher level “places” in the hierarchy. Depending on the context, a “place” might be a health facility and the “people” who get created at that level might be nurses. Most often for CHWs, these “places” are families.
```
Comment (abbyad) [inline content/en/apps/features/contacts/_index.md:20]: > Most often for CHWs, these “places” are families.

For CHWs there are still multiple level, multiple type of "places". For deployments with CHWs using mobile apps the lowest level of the hierarchy is generally families or households. Here is an attempt to clarify: 

```suggestion
“Places” is the generic name that represents a level in the [hierarchy]({{< ref "apps/concepts/hierarchy" >}}). “People” belong to “places”, and “places” belong to other higher level “places” in the hierarchy. A “place” could be a geographic area, like a district with the "people" associated to it being health officers. A "place" could also be a structure in the health system, such as a health facility, and the "people" associated to it being nurses. In deployments with CHWs, the lowest "place" in the hierarchy often represents individual households or families, and the individual members of that household are the "people" associated to it. 
```

File: content/en/apps/features/contacts/_index.md [hunk 2]
```diff
@@ -10,19 +10,16 @@ relatedContent: >
...
+“Places” is the generic name that represents a level in the [hierarchy]({{< ref "apps/concepts/hierarchy" >}}). “People” belong to “places” and “places” belong to other higher level “places” in the hierarchy. Depending on the context, a “place” might be a health facility and the “people” who get created at that level might be nurses. Most often for CHWs, these “places” are families. 
 
-Depending on the context, a “place” might be a health facility and the “people” who get created at that level might be nurses. Most often for CHWs, these “places” are families. 
-
-Users can access their “people” and “places” from the People tab. 
+Users can access their “people” and “places” from the People tab. The permissions set for your role and your placement in the hierarchy will determine which contacts you’re able to see. Advanced configuration options are available for a specific off-line user role to manage what [level of contact data]({{< ref "apps/guides/performance/replication#contact-depth" >}}) is copied to their device.
```
Comment (abbyad) [inline content/en/apps/features/contacts/_index.md:22]: Bold UI element:

```suggestion
Users can access their “people” and “places” from the **People** tab. The permissions set for your role and your placement in the hierarchy will determine which contacts you’re able to see. Advanced configuration options are available for a specific offline user role to manage what [level of contact data]({{< ref "apps/guides/performance/replication#contact-depth" >}}) is downloaded and stored on their device.
```

File: content/en/apps/concepts/users.md [hunk 3]
```diff
@@ -32,7 +32,9 @@ Online roles are for users who need access to a lot of data and need to maintain
 
 ### Offline Users
 
-Offline roles are for users who need to be able to access data on the go in the field, don’t need to maintain the system, and don’t have a reliable internet connection. All the data they have access to will be synced to their device.
+Offline roles are for users who need to be able to access data on the go in the field, don’t need to maintain the system, and don’t have a reliable internet connection. All the data they have access to will be synced to their device. 
+
+{{% alert title="Note" %}} Advanced configuration options are available for a specific off-line user role to manage what [level of data]({{< ref "apps/guides/performance/replication" >}}) is synced to their device. {{% /alert %}}
```
Comment (abbyad) [inline content/en/apps/concepts/users.md:37]:
```suggestion
{{% alert title="Note" %}} Advanced configuration options are available for a specific offline user role to manage what [level of data]({{< ref "apps/guides/performance/replication" >}}) is synced to their device. {{% /alert %}}
```

File: content/en/apps/concepts/users.md [hunk 4]
```diff
@@ -32,7 +32,9 @@ Online roles are for users who need access to a lot of data and need to maintain
 
 ### Offline Users
 
-Offline roles are for users who need to be able to access data on the go in the field, don’t need to maintain the system, and don’t have a reliable internet connection. All the data they have access to will be synced to their device.
+Offline roles are for users who need to be able to access data on the go in the field, don’t need to maintain the system, and don’t have a reliable internet connection. All the data they have access to will be synced to their device.
```
Comment (abbyad) [inline content/en/apps/concepts/users.md:35]:
```suggestion
Offline roles are for users who need to be able to access data on-the-go in the field and don’t have a reliable internet connection. All the data they have access to will be synced to their device. System administrators cannot be offline users as they won't have access to the app management tools offline.
```

**Comment by MaxDiz** [comment]
Your changes looks good @abbyad , thanks! I accepted, but looks like I still need your sign-off to merge

**Pull request merged**